### PR TITLE
Migrating to React version 16.0.0

### LINKED ISSUE #1: Upgrade to React 16.0.0
I tried to use this package with the newly release React 16.0.0 and this error pops up:

image

### PR DESCRIPTION
**Pull request by ioan-ungurean**
Migrating to React version 16.0.0 Fixes #1

### CODE REVIEW

File: test/test-duo.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+import Adapter from 'enzyme-adapter-react-16';
+
 import React from 'react';
 import sinon from 'sinon';
 import test from 'tape';
 
 import Duo from '../src/duo';
 import duoWeb from '../lib/duo-web';
 
+Enzyme.configure({adapter: new Adapter()});
```
[the remark this answers is not in the document]
Comment (ioan-ungurean) [reply test/test-duo.js]: @LINKIWI I made a new PR and addressed all your comment. I added a new config file for enzyme adaptor.

[the remark this answers is not in the document]
Comment (ioan-ungurean) [reply test/test-duo.js]: Please let me know if it's ok.

**Comment by LINKIWI** [comment]
Looks good, thanks!

**Comment by LINKIWI** [comment]
This has been published under the first major semver, `1.0.0`.

**Pull request merged**